packages/dd-trace/src/plugins/util/web.js

```diff
@@ -37,6 +37,17 @@ const HTTP2_HEADER_PATH = ':path'
 const contexts = new WeakMap()
 const ends = new WeakMap()
 
+const TracingPlugin = require('../tracing')
+
+function startSpanHelper (tracer, name, options, traceCtx, config = {}) {
+  return TracingPlugin.prototype.startSpan.call(
```

Comment:
This seems like an anti-pattern, prototype methods are meant to become available on instances of the class, not be accessed directly.

Reply:
Changed the implementation to create a new instance of tracing plugin for web to use.